server/tsconfig.test.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "noUnusedLocals": false,
```

Comment:
Are those deactivated warnings or something? :thinking:

Reply:
These are more than warnings: it stops completely the compilation if these warnings are found.
Because in the server part we are using ANTLR-generated files that creates some of those warnings, I preferred to deactivate some.

In this file, we are using the same options as in production, except the output directory.